test/containerbuild.py

```diff
@@ -24,5 +24,9 @@ def container_to_build_ref():
     # making "image_type" an "image" tuple (type, container_ref_to_test)
     return os.getenv(
         "BIB_TEST_BOOTC_CONTAINER_TAG",
-        "quay.io/centos-bootc/fedora-bootc:eln",
+        # using this tag instead of ":eln" until
+        #  https://github.com/CentOS/centos-bootc/issues/184 and
+        #  https://github.com/osbuild/bootc-image-builder/issues/149
+        # are fixed
+        "quay.io/centos-bootc/fedora-bootc:ed19452a30c50900be0b78db5f68d9826cc14a2e402f752535716cffd92b4445",
```

Comment:
Yeah, we definitely need auto-rotating datestamped tags...

Reply:
If/when we get those, let's also have a regular auto-update job here so that changes in the base container are handled in isolated PRs.